nss-day-cohort-30/bangazon-site-suave-salmon
# Mo working branch

**Fixes:**
Order history view

**Changes proposed in this request & reasons behind organizational or architectural decisions:**
User can now navigate to profile, click on the order history hyperlink, and view their completed orders with a list of products associated with that order, and a total price. 

**Steps to test:**
- git fetch --all
- git checkout MoWorkingBranch
- git pull MoWorkingBranch
- navigate to the profile, click on order history, If your user has completed orders they should appeal

**System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):**

none


Link to feature ticket:
https://github.com/nss-day-cohort-30/bangazon-site-suave-salmon/issues/18

@suave-salmon
